.github/workflows/linting.yaml

```diff
@@ -24,7 +24,7 @@ jobs:
         uses: actions/setup-python@v4
         with:
           python-version: ${{ env.DEFAULT_PYTHON }}
-          cache: 'poetry'
+          cache: "poetry"
```

Comment:
Why don't we drop the quotes altogether?

Reply:
That's something for later